### ISSUE TITLE
Recording Your Own Demonstrations : Not Working

### ISSUE THREAD

**Issue by NajibRaji** [stanfordnlp/miniwob-plusplus#3]
Hi ! I'm a RL beginner and I tried to record a demonstration on the click-test.html
When I write the path : file:///path/to/miniwob-plusplus/html/miniwob/click-test.html, the page is rendered correctly.
Then as explained in the instructions, I created a directory named 'out' and launched the record.py script.

My problem is that when I add '?record=true' to the path of the website, I get 'ERR_FILE_NOT_FOUND' in my browser (Google Chrome).

Thank you in advance for your answer.

**Comment by ppasupat** [comment]
Hi! Somehow Chrome converted the `?` in `?record=true` into an escape sequence. Try pasting the whole URL in Chrome's location bar instead:
```
file:///path/to/miniwob-plusplus/html/miniwob/click-test.html?record=true
```

**Comment by NajibRaji** [comment]
Hi! Thank you for your quick response. The problem was as you said the conversion of '?' into an escape sequence. I tried to copy past the whole URL in Chrome's location bar but it still didn't work. 

I then tried to run a node http-server and it worked fine (the '?' was not escaped anymore).

PS : This is not directly related with this issue but I've seen there was a plateform for WoB on openAI but all the links I find are dead links. Is there still a WoB benchmark on openAI ?

Thank you very much for your time.
